translations/en-us.json

```diff
@@ -63,6 +82,13 @@
   "scoped_knowledge_base_search_description": "Search results are confined to the category the user is in",
   "scoped_knowledge_base_search_label": "Scoped search in Knowledge Base",
   "search_group_label": "Search settings",
+  "header_page_group_label": "Header settings",
+  "user_info_label": "User info",
+  "user_info_description": "Show user info in header",
+  "mit_link_label": "MIT link",
+  "mit_link_description": "Specify the URL for MIT image in header",
```

Comment:
Is this being used?

Reply:
Removed this as well since the related setting is not being utilized.